go/libkb/api_stub_html.go

```diff
@@ -0,0 +1,577 @@
+package libkb
+
+var htmlAPIStubs = map[string]string{
```

Comment:
might as well try to integrate `vcr` in client codebase, what do you think @patrickxb ?

Reply:
sure, it's already used in client for stellar.

but whatever's easiest.  if this is done, then that's fine...

(also, i know we already have/had this somewhere for github and maybe twitter...maybe it was deleted)

Reply:
oh, it's this api_stub.go file.  

@zapu i say just keep using this api_stub system for now and move on to better things...

Reply:
sounds good. I didn't know it was used in client, I couldn't find it. I thought it was only in stellard and stellarnet

Reply:
@patrickxb merge ok? we don't have to wait for Max if you think this solution is fine.